railties/test/generators/app_generator_test.rb

```diff
@@ -925,6 +889,15 @@ def test_css_option_with_cssbundling_gem_does_not_force_jsbundling_gem
     assert_gem "importmap-rails"
   end
 
+  def test_default_generator_executes_all_rails_commands
+    run_generator_and_bundler [destination_root]
+
+    expected_commands = [
+      "credentials:diff --enroll", "importmap:install", "turbo:install stimulus:install"
+    ]
+    assert_equal expected_commands, @rails_commands
+  end
```

Comment:
I don't think this test is necessary.  [`test_generator_configures_decrypted_diffs_by_default`](https://github.com/rails/rails/blob/121e0ad015c74eaaca82c6c9a85f5b2e0672ca0b/railties/test/generators/app_generator_test.rb#L531-L534) already asserts the result of `credentials:diff --enroll`, and the tests you've modified cover the other commands.  Additionally, this test is brittle.  We would have to update it whenever we add or remove and `rails_command` calls in the app generator.

Reply:
The idea was to make sure that inserting a random `rails_command "lol"` somewhere will trip something somewhere in the test suite, and not only when you run it live, same goes for the empty test case.